temporalcli/commandsmd/commands.md

```diff
@@ -911,6 +911,7 @@ Use the options listed below to change reset behavior.
 * `--build-id` (string) - Only used if type is BuildId. Reset the first workflow task processed by this build id. Note that by default, this reset is allowed to be to a prior run in a chain of continue-as-new.
 * `--query`, `-q` (string) - Start a batch reset to operate on Workflow Executions with given List Filter.
 * `--yes`, `-y` (bool) - Confirm prompt to perform batch. Only allowed if query is present.
+* `--rps` (float) - Limit batch's requests per second. Only allowed if query is present.
```

Comment:
Can you update the top of this file to include this new data type in the bullet that lists the data types supported? It's our admittedly-informal "spec".